federatedscope/core/auxiliaries/dataloader_builder.py

```diff
@@ -6,36 +8,52 @@
     Dataset = object
 
 
-def get_dataloader(dataset, config):
+def get_dataloader(dataset, config, split='train'):
```

Comment:
Please provide docstring to describe the purpose of each arg. I cannot easily figure it out.